### DISCUSSION TITLE
Create animation to explain key concepts of Kedro?

### DISCUSSION THREAD

**Discussion by noklam** [kedro-org/kedro#4491] [Idea]
## Description

Not really a priority, I save this for Hackathon idea as I have been wanting to experiment with https://github.com/3b1b/manim for long time.

Something like this https://twitter.com/i/status/1363494433715552259

Ideas for candidates?
- Kedro Node & Data Catalog & catalog.yml 
  - For example, explains how Node requests data from DataCatalog which reads `catalog.yml`, this is often hard to understand for beginner [when they try to create custom dataset](https://stackoverflow.com/questions/76766933/how-to-create-kedro-catalog-entry-custom-dataset-init-method-with-values-from)
- KedroSession & hooks
- Kedro run lifecycle
- Hook execution orders
- Any idea?

**Comment by stichbury** [comment]
This is a great idea. I'm wondering if this kind of animation fits into docs, decks or devrel, or even something for a blog post. It's kind of a general thing that we should have more graphical explanation to sit alongside our written content. I [have a ticket](https://github.com/kedro-org/kedro/issues/1718) for docs in particular so will make sure to consider this tool as a way to make animations where they are useful & relevant. 

Couple of ideas for content based on https://github.com/kedro-org/kedro/issues/1718 include `kedro run` lifecycle and hooks execution order.

**Comment by noklam** [comment]
@stichbury I am glad you like the idea, I find this initially from https://twitter.com/_ScottCondron/status/1363494433715552259?t=YOucme8E9_m8GDbVBYCG4g&s=19

I would like to see it to be shared on platform/social media. We talked about the idea of some weekly Kedro tips before but we never did it. Creating some small modular content is quite useful even if it's not most up to date.

**Comment by stichbury** [comment]
Definitely a good idea. We don't really have an audience on social media though, so it's about getting this in front of people in as many ways as we can. It would be interesting to explore with @NeroOkwa if there's scope in Viz for a sort-of advert slot where we put marketing messages and links to useful content like this, but for now, let's keep this in docs as a primary driver to create the content, and then worry about how we distribute it to maximum numbers.

**Comment by noklam** [comment]
Moving this out from `inbox` since I don't think we have capacity for this in short term.

**Comment by astrojuanlu** [comment]
A nice idea whose time hasn't come yet. Turning this into a discussion.
